tests/browser/report-generation.test.ts

```diff
@@ -0,0 +1,50 @@
+import { describe, beforeAll, it, expect as globalExpect } from 'vitest';
+import {
+  bundlers,
+  type BundlerValues,
+  cleanup,
+  skipCleanup,
+  scopeToFixture,
+} from '@sku-private/testing-library';
+import fs from 'node:fs';
+
+const timeout = 50_000;
+
+const { sku, fixturePath } = scopeToFixture('report-generation');
+
+describe('report-generation', () => {
+  describe.sequential.for(bundlers)('bundler %s', (bundler) => {
+    describe('build', async () => {
+      const args: BundlerValues<string[]> = {
+        vite: ['--config', 'sku.config.vite.ts', '--experimental-bundler'],
+        webpack: [],
+      };
+
+      beforeAll(async () => {
+        // Clean up any existing report folder
+        const reportDir = fixturePath('report');
+        if (fs.existsSync(reportDir)) {
+          fs.rmSync(reportDir, { recursive: true, force: true });
+        }
```

Comment:
using `await fs.rm(...)` won't fail if the file doesn't exits so you could skip checking existence 

```suggestion
        import fs from 'node:fs/promises';
        // ...
        await fs.rm(reportDir, { recursive: true, force: true });
```